firmware-binaries/examples/smoltcp_client/src/main.rs

```diff
@@ -42,6 +42,8 @@ const CHUNK_SIZE: usize = 4096;
 const SERVER_IP: IpAddress = IpAddress::v4(10, 0, 0, 1);
 const SERVER_PORT: u16 = 1234;
 
+// See https://github.com/bittide/bittide-hardware/issues/681
+#[allow(static_mut_refs)]
```

Comment:
I have Opinions™ on how this should be removed, but for now at least this can stay.